test/check-machines-networks

```diff
@@ -219,7 +229,7 @@ class TestMachinesNetworks(machineslib.VirtualMachinesCase):
                         for xfo in self.xfail_objects:
                             if "range" in xfo:
                                 b.wait_in_text(
-                                    f"#create-network-dialog .create-network-{xfo} .pf-v5-c-form__helper-text .pf-v5-c-helper-text__item.pf-m-error",
+                        f"#create-network-dialog .create-network-{xfo} .pf-v5-c-form__helper-text .pf-v5-c-helper-text__item.pf-m-error",
```

Comment:
This is too outdented now, and IMHO worse than before --  better split up the string to two lines